tools/lang_batch_translate.py

```
@@ -0,0 +1,256 @@
+
+    translator = TRANSLATOR_CACHE.get(locale)
+    if locale not in TRANSLATOR_CACHE:
+        translator = get_translator(locale)
+        TRANSLATOR_CACHE[locale] = translator
+
+    if translator is None:
+        return {text: text for text in texts}
+
+    try:
+        translated_values = translator.translate_batch(texts)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)